README.md

```diff
@@ -1,86 +1,163 @@
+
+Simply pulling `linuxserver/letsencrypt` should retrieve the correct image for your arch, but you can also pull specific arch images via tags.
+
+The architectures supported by this image are:
+
+| Architecture | Tag |
+| :----: | --- |
+| x86-64 | amd64-latest |
+| arm64 | arm64v8-latest |
+| armhf | arm32v6-latest |
 
-[![letsencrypt](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/le-logo-wide.png)][appurl]
 
 ## Usage
 
+Here are some example snippets to help you get started creating a container.
+
+### docker
+
 ```
 docker create \
-  --cap-add=NET_ADMIN \
   --name=letsencrypt \
-  -v <path to data>:/config \
-  -e PGID=<gid> -e PUID=<uid>  \
-  -e EMAIL=<email> \
-  -e URL=<url> \
-  -e SUBDOMAINS=<subdomains> \
-  -e VALIDATION=<method> \
-  -p 80:80 -p 443:443 \
-  -e TZ=<timezone> \
+  --cap-add=NET_ADMIN \
+  -e PUID=1001 \
+  -e PGID=1001 \
+  -e TZ=Europe/London \
+  -e URL=yourdomain.url \
+  -e SUBDOMAINS=www, \
+  -e VALIDATION=http \
+  -e TZ=Europe/London \
+  -e DNSPLUGIN=cloudflare `#optional` \
+  -e DUCKDNSTOKEN=<token> `#optional` \
+  -e EMAIL=<e-mail> `#optional` \
+  -e DHLEVEL=2048 `#optional` \
+  -e ONLY_SUBDOMAINS=false `#optional` \
+  -e EXTRA_DOMAINS=<extradomains> `#optional` \
+  -e STAGING=false `#optional` \
+  -p 443:443 \
+  -p 80:80 `#optional` \
+  -v </path/to/appdata/config>:/config \
+  --restart unless-stopped \
   linuxserver/letsencrypt
 ```
 
-## Parameters
 
-`The parameters are split into two halves, separated by a colon, the left hand side representing the host and the right the container side.
-For example with a port -p external:internal - what this shows is the port mapping from internal to external of the container.
-So -p 8080:80 would expose port 80 from inside the container to be accessible from the host's IP on port 8080
-http://192.168.x.x:8080 would show you what's running INSIDE the container on port 80.`
+### docker-compose
 
+Compatible with docker-compose v2 schemas.
+
+```
+---
+version: "2"
+services:
+  letsencrypt:
+    image: linuxserver/letsencrypt
+    container_name: letsencrypt
+    cap_add:
+      - NET_ADMIN
+    environment:
+      - PUID=1001
+      - PGID=1001
+      - TZ=Europe/London
+      - URL=yourdomain.url
+      - SUBDOMAINS=www,
+      - VALIDATION=http
+      - TZ=Europe/London
+      - DNSPLUGIN=cloudflare #optional
+      - DUCKDNSTOKEN=<token> #optional
+      - EMAIL=<e-mail> #optional
+      - DHLEVEL=2048 #optional
+      - ONLY_SUBDOMAINS=false #optional
+      - EXTRA_DOMAINS=<extradomains> #optional
+      - STAGING=false #optional
+    volumes:
+      - </path/to/appdata/config>:/config
+    ports:
+      - 443:443
+    ports:
```

Comment:
Ports cannot be defined twice. Compose will not throw an error, but only the bottom-most definition will be used, so in this case 443 would not be used.

Suggested:
```yaml
    ports:
      - 443:443
      - 80:80 #optional
```